commit 867fc7f3d2a2fab65fb6ded29e0c621592f91c69 (refs/changes/73/2073/2)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2017-06-12T20:39:36+00:00 (2 years, 4 months ago)
    
    CNS-205 CNS should pull in just sprintf-js@1.0.3
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>
    Approved by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/package.json b/package.json
index ec59e35..550cd87 100644
--- a/package.json
+++ b/package.json
@@ -40,7 +40,7 @@
     "restify-clients": "1.5.0",
     "restify-errors": "4.3.0",
     "sdc-scripts": "git+https://github.com/joyent/sdc-scripts.git#d01c308",
-    "sprintf-js": "^1.0.3",
+    "sprintf-js": "1.0.3",
     "triton-tags": "^1.1.2",
     "uuid": "2.0.1",
     "vasync": "1.6.3"
